requirements.txt

```diff
@@ -11,4 +11,6 @@ sphinxcontrib-napoleon
 sphinx-rtd-theme
 stackstac
 folium
-boto3
\ No newline at end of file
+boto3
+selenium
+Pillow
```

Comment:
I also had to 
```
pip3 install webdriver_manager
```
```suggestion
webdriver_manager
```

Reply:
I got a missing dependency error trying to load the helper files when it tries to run `from selenium import webdriver`